packages/tools/src/commands/create-plugin/templates.ts

```diff
@@ -175,7 +175,7 @@ export const getFiles = (context: Context): File[] => {
                             "@dd/core": "workspace:*"
                         },
                         "devDependencies": {
-                            "typescript": "${pkg.devDependencies.typescript}",
+                            "typescript": "${pkg.devDependencies.typescript}"
```

Comment:
This is just a fix for the `yarn cli create-plugin` command that would produce a faulty `package.json` breaking any `yarn` run afterward.